src/main/java/frc/robot/poseestimator/helpers/ObservationCountHelper.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.poseestimator.helpers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+public class ObservationCountHelper<T> {
+
+	private final Supplier<List<T>> observationSupplier;
+	private final int maxCount;
+	private List<T> stackedObservations;
+	private int observationsCount;
```

Comment:
This is *not* the observationCount but the count of times that the poseestimator dumped data to the countHelper (e.g., for 3 cameras, each of those seeing April Tags the entire time, this var would be 10 while the actual observation count would be 30).

Reply:
This was not resolved yet marked as such